dnf-behave-tests/features/fail-safe.feature

```diff
@@ -0,0 +1,564 @@
+       | dnf-ci-fedora-modular-updates | nodejs        | 8 [d]     | development, minimal, default [d] |
+       | dnf-ci-fedora-modular-updates | nodejs        | 10        | development, minimal, default [d] |
+       | dnf-ci-fedora-modular-updates | nodejs        | 11        | development, minimal, default     |
+       | dnf-ci-fedora-modular-updates | nodejs        | 12        | development, minimal, default     |
+       | dnf-ci-fedora-modular-updates | postgresql    | 9.6 [d]   | client, server, default [d]       |
+       | dnf-ci-fedora-modular-updates | postgresql    | 10 [e]    | client, server, default           |
+       | dnf-ci-fedora-modular-updates | postgresql    | 11        | client, server, default           |
+
+
+@bz1616167
+@bz1623128
+Scenario Outline: When modulemd is not available, 'module list' shows fail-safe modules
+   When I use the repository "dnf-ci-fedora-modular-updates"
+   When I execute dnf with args "module enable postgresql:10"
+   Then the exit code is 0
+    And Transaction contains
+        | Action                    | Package                |
+        | module-stream-enable      | postgresql:10          |
+    And modules state is following
+        | Module         | State     | Stream    | Profiles  |
+        | nodejs         | enabled   | 5         |           |
+        | postgresql     | enabled   | 10        |           |
+  Given I execute step "<step>"
+    And I execute dnf with args "clean all"
+   When I execute dnf with args "module list"
+   Then the exit code is 0
+    And module list is
+        | Repository                    | Name          | Stream    | Profiles                      |
+        | dnf-ci-fedora-modular-updates | nodejs        | 8         | development, minimal, default |
+        | dnf-ci-fedora-modular-updates | nodejs        | 10        | development, minimal, default |
+        | dnf-ci-fedora-modular-updates | nodejs        | 11        | development, minimal, default |
+        | dnf-ci-fedora-modular-updates | nodejs        | 12        | development, minimal, default |
+        | dnf-ci-fedora-modular-updates | postgresql    | 9.6       | client, server, default       |
+        | dnf-ci-fedora-modular-updates | postgresql    | 10 [e]    | client, server, default       |
+        | dnf-ci-fedora-modular-updates | postgresql    | 11        | client, server, default       |
+        | @modulefailsafe               | nodejs        | 5 [e]     | development, default, minimal |
+
+Examples:
+    | step                                                                                       |
+    | Given I disable the repository "dnf-ci-fedora-modular"                                     |
+    | Given I delete directory "/temp-repos/dnf-ci-fedora-modular"                               |
+    | Given I delete file "/temp-repos/dnf-ci-fedora-modular/repodata/*modules.yaml*" with globs |
```

Comment:
Is this step supposing to disable possibility to get a valid repodata?

Reply:
This is supposed to prevent getting valid modular metadata.